### DISCUSSION TITLE
Account Issue

### DISCUSSION THREAD

**Discussion by JaniceBPC** [SauceLLC/sauce4strava#198] [General]
I did sign up am paying $5/mo, however When selecting the Sauce Fitness & Tracking I am still getting  the Sauce Welcome Splash screen

**Comment by mayfield** [comment]
Hi @JaniceBPC Can you double check that you have your browser linked with patreon using this link: 

  https://sauce.llc/patreon-oauth

After that's connected then the Sauce Performance page should have a big red "Activate.." button instead of the Patreon button.
